app/views/warehouse_reports/hud_chronics/_sort.haml

```diff
@@ -11,8 +11,8 @@
           - else
             - title = "<span class='icon-checkbox-unchecked'></span> #{opt[:title]}".html_safe
           %li
-            - # FIXME replace permit! with a whitelist and remove from brakeman.ignore
-            = link_to title, params.permit!.merge!(direction: opt[:direction], column: opt[:column])
+            = link_to({direction: opt[:direction], column: opt[:column]}) do
```

Comment:
This will also lose other filter options

Reply:
The HUD Chronics report only has Sort.